developer/src/tike/xml/help/contexthelp.xml

```diff
@@ -387,6 +387,155 @@
     </Control>
   </Form>
 
+  <Form Name="context/options">
+    <Control Name="pages" Title="Options Dialog">
+      <p>The following options are configurable in TIKE in the General tab.</p>
+    </Control>
+
+    <Control Name="chkShowStartupDialog" Title="General tab">
+      <p></p>
+    </Control>
```

Comment:
I see a few examples where you haven't added content -- is this because the content is missing on help.keyman.com? We should aim to update both in that situation!

Reply:
Yes, there's no info on help.keyman.com.

Reply:
What can I do with missing contents?

Reply:
I think we need to write some content. Do you want to do that with @Meng-Heng or would you prefer I write something up?

Reply:
I would like you to write for us cause it would be more accurate. Also, for some functions that no longer appear in 17.0 like `chkShowStartupDialog` and `chkAllowMultipleInstances`, can I just delete it?

Reply:
> I would like you to write for us cause it would be more accurate.

OK. I will work on it when I get some time!

> Also, for some functions that no longer appear in 17.0 like `chkShowStartupDialog` and `chkAllowMultipleInstances`, can I just delete it?

Yes

Reply:
Noted! I already removed it.